Core/src/org/sleuthkit/autopsy/centralrepository/persona/PersonasTopComponent.java

```diff
@@ -276,6 +277,7 @@ private void initComponents() {
         editBtn = new javax.swing.JButton();
         deleteBtn = new javax.swing.JButton();
         jSeparator1 = new javax.swing.JSeparator();
+        jScrollPane2 = new javax.swing.JScrollPane();
```

Comment:
Please give the scroll panes more descriptive names.  Please fix jSplitePane1 while you are at it as well.